### PR TITLE
Better UI integration with FlightRadar24 in addition to FlightAware

### PR DESCRIPTION
**Pull request by e2jk**
PR for #44.

### CODE REVIEW

**Comment by e2jk** [comment]
First version of the PR, for now only containing the changes to add the FR24 logo and modify the page name to a neutral name.
I'd be interested in hearing from you how you see this, if this approach suits you I will continue modifying to add the flight links to FR24 and to the radar's public pages on FA and FR24.

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

File: patch/flightradar24.patch [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ }
+
++.flightradar24Logo {
++    padding-top: 7px;
++    padding-bottom: 6px;
++    padding-left: 20px;
++    padding-right: 20px;
++    width: 121px;
++    height: 25px;
++    box-shadow:inset -12px 0 12px -12px #000000;
```
Comment (Thom-x) [inline patch/flightradar24.patch:61]: We need to remove `box-shadow`

image

**Comment by Thom-x** [comment]
@e2jk finally the PR is available with tag [PR-45](https://hub.docker.com/layers/139721954/thomx/fr24feed-piaware/PR-45/images/sha256-0256348537edac1b7964831cc855322c944c7e266de3b6b6d85113b85a2840f6?context=explore&tab=layers)

**Comment by Thom-x** [comment]
It should look like this now:
image

**Comment by e2jk** [comment]
I've updated my container, and I confirm it works as expected (with the ugly `box-shadow`, simply copied from the FA logo).
Thanks for the update to the CSS style, your screenshot indeed looks better.

So, are you OK continuing in this direction (patch file to update the UI)? If so, I'll continue working on it.

(Regarding the link to the radar's page on the FA website, I'd first create a pull request to upstream, so that adding the link to the  FR24 page is added to a consistent place)

**Comment by Thom-x** [comment]
> So, are you OK continuing in this direction (patch file to update the UI)?
Yes, let's continue.

**Comment by e2jk** [comment]
ça roule!

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
Ok, new image pushed, working as expected 👍

**Comment by Thom-x** [comment]
@e2jk Should we merge the PR or you have more work to do ?

**Comment by e2jk** [comment]
Yes, there is more to do, but I'm not sure when I'll get to it. Also, for the links to the individual radar's stats pages on both sites, I actually want to send a PR upstream for the link to FlightAware, and once that one is merged work on getting the FR24 link included next to it. So it might take a bit of time for the FlightAware patch to be accepted and put in a release that we can work from here...

The changes already made are stand-alone, so you can already merge these and I'll send the other changes via another PR.

**Pull request merged**